Intro carpenter (#2425)
giantswarm/docs: src/content/tutorials/fleet-management/cluster-management/cluster-scaling/_index.md

  principal:
    parent: tutorials-fleet-management-clusters
    identifier: tutorials-fleet-management-clusters-scaling
last_review_date: 2024-12-13
owner:
  - https://github.com/orgs/giantswarm/teams/team-rocket
  - https://github.com/orgs/giantswarm/teams/team-phoenix
user_questions:
  - How can I scale my workload clusters on Giant Swarm using Karpenter and cluster autoscaler?
---

> In Gs we combine cluster autoscaler and karpenter to have the better scaling possible in your workload clusters. This tutorial will guide you through the configuration and management of the scaling of your workload clusters on Giant Swarm.

Reviewer: ```suggestion
At Giant Swarm, we combine the [cluster autoscaler](https://github.com/kubernetes/autoscaler) and [karpenter](https://karpenter.sh/) to reach optimal scaling of your workload clusters. This tutorial will guide you through the configuration and management of both.
```
